src/components/Cards/ExpertList.js

```diff
@@ -33,54 +40,62 @@ const ImageWrapper = styled.div`
 const DetailPreview = styled.div`
```

Comment:
I think it would be be more semantic to work with a list again. Then you can reduce the use of `div` elements. Just a recommendation.